### ISSUE TITLE
You called getEntry on the Space API. Since version 4.0.0 the Space API and its methods are deprecated.

### ISSUE THREAD

**Issue by brianlmacdonald** [contentful/field-editors#1171]
I'm seeing this warning in the console when using these field editors- they seem to be using parts of the app-sdk that are deprecated.

![Screen Shot 2022-06-30 at 10 11 01 AM](https://user-images.githubusercontent.com/23248886/176699168-b02c6592-8ea9-4d6b-b5d0-4d730ad0abc6.png)

Are there plans to switch out the sdk as a client for the cma instead?

**Comment by z0al** [comment]
Yes, this is on our radar. We will migrate to the CMA client once a few blockers are resolved. We are in touch with the team responsible for the app-sdk internally to resolve the matter.
